index.html

```diff
@@ -739,7 +739,7 @@ <h3>The <dfn><code>pointercancel</code> event</dfn></h3>
                     <li>The user agent has determined that a pointer is unlikely to continue to produce events (for example, because of a hardware event).</li>
                     <li>After having fired the <code>pointerdown</code> event, if the pointer is subsequently used to manipulate the page viewport (e.g. panning or zooming).
                      <div class="note">User agents can trigger panning or zooming through multiple pointer types (such as touch and pen),
-                     and therefore the start of a pan or zoom action may result in the cancellation of various pointers, including pointers with different pointer types.
+                     and therefore the start of a scroll or zoom action may result in the cancellation of various pointers, including pointers with different pointer types.
```

Comment:
I think we decided we should go with pan consistently even though the two should be interchangeable?

Reply:
odd, i thought i had caught all instances of where i made a change and reverted them...good spot (here and in the other place)